README.md

```diff
@@ -52,14 +50,14 @@
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="">N/A</a></li>
+    <li><a href="">ROR</a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li>N/A</li>
+    <li>POSTGRESSQL</li>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you rename your database because some letters are capitalized and it contains only `S` last it should be a link to the PostgreSQL website :+1: After making changes it should look like below :arrow_down:

```md
<li><a href="https://www.postgresql.org">PostgreSQL</a><li>
```

Reply:
HI @HAMMAS-SALEEM ,
Thank you.
Done